tracker/bool.go

```diff
@@ -67,6 +69,7 @@ func (m *IsRecording) Bool() Bool  { return Bool{m} }
 func (m *IsRecording) Value() bool { return (*Model)(m).recording }
 func (m *IsRecording) setValue(val bool) {
 	m.recording = val
+	// this toggles the NoteEditor because it does not get updated real-time
```

Comment:
damn, I forgot this. it was somewhat related to the issue because at first, I did not understand why the panel was hidden while recording. I'll remove my comment.